packages/dds/merge-tree/src/test/obliterate.partialLength.spec.ts

```diff
@@ -5,21 +5,23 @@
 
 import { strict as assert } from "node:assert";
 
+import { NonCollabClient } from "../constants.js";
 import { MergeTreeDeltaType } from "../ops.js";
 
 import { TestClient } from "./testClient.js";
 import {
 	insertText,
-	obliterateRange,
 	useStrictPartialLengthChecks,
 	validatePartialLengths,
 } from "./testUtils.js";
 
 describe("obliterate partial lengths", () => {
 	let client: TestClient;
 	let refSeq: number;
-	const localClientId = 17;
```

Comment:
this suite was doing really strange things before that don't really make much sense, and the validation function it used didn't really make sense either, so a lot of the logic here is rewritten. We're still testing the same core scenarios though and comparing values obtained from partial lengths by the values obtained from summing all segment lengths.